### PR TITLE
Feed

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Implements UI components for the feed. 

- Proposal Card
- Hero banner
- Proposal Filters

Not responsive yet. Will be working on it in the coming PRs.
Screen Shot 2020-10-12 at 8 38 42 PM

### CODE REVIEW

File: src/components/Feed/ProposalFooter.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            canSupport={inactiveTokens.gt(0)}
+            proposal={proposal}
+            onThumbsUp={handleThumbsUp}
+            onThumbsDown={handleThumbsDown}
+          />
+          <div>
+            {supportersCount} Supporter{supportersCount === 1 ? '' : 's'}
+          </div>
+        </div>
+        <div>Status : {proposal.status}</div>
```
Comment (rperez89) [inline src/components/Feed/ProposalFooter.js:71]:
```suggestion
        <div>Status: {proposal.status}</div>
```

File: src/components/FilterSidebar/FilterSidebar.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    proposalSupportFilter,
+    proposalTypeFilter,
+    onClearFilters,
+    onStatusFilterChange,
+    onSupportFilterChange,
+    onTypeFilterChange,
+  }) => {
+    const theme = useTheme()
+    const { account } = useWallet()
+    const supportFilterDisabled = proposalStatusFilter > 1 || !account
```
Comment (rperez89) [inline src/components/FilterSidebar/FilterSidebar.js:22]: we might want to add a comment here saying what is `1`?

File: src/components/FilterSidebar/ListFilter.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <div
+      onClick={handleOnClick}
+      css={`
+        margin-bottom: ${1 * GU}px;
+        color: ${theme.contentSecondary};
+
+        ${selected === index
+          ? `
+          color:${theme.content};
```
Comment (rperez89) [inline src/components/FilterSidebar/ListFilter.js:36]:
```suggestion
          color:${theme.content}
```

File: src/components/FilterSidebar/ListFilter.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        `
+          : 'cursor:pointer;'};
```
Comment (rperez89) [inline src/components/FilterSidebar/ListFilter.js:38]:
```suggestion
          : 'cursor:pointer'};
```

File: src/components/Header/Header.js [hunk 5]
```diff
@@ -18,74 +18,93 @@ function Header({ compact }) {
...
+                external={false}
+                css={`
+                  text-decoration: none;
+                  color: ${theme.contentSecondary};
+                `}
+              >
+                Home
+              </Link>
+              <Link
+                href="https://honeyswap.org/#/swap?inputCurrency=0x71850b7e9ee3f13ab46d67167341e4bdc905eef9"
```
Comment (rperez89) [inline src/components/Header/Header.js:63]: we might want to add this to the endpoints file

File: src/components/ProposalIcon.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { GU } from '@1hive/1hive-ui'
+import { ProposalTypes } from '../types'
+
+function ProposalIcon({ type }) {
+  let color
+  if (type === ProposalTypes.Decision) {
+    color = '#FFC3AB'
```
Comment (rperez89) [inline src/components/ProposalIcon.js:8]: we don't have those colors on the ui right?

**Review by rperez89 — COMMENTED** [review]
MAAAAAAN!!!!

File: src/components/FilterSidebar/FilterSidebar.js (hunk 2 above)
Comment (fabriziovigevani) [reply src/components/FilterSidebar/FilterSidebar.js:22]: Ah yes, we should probably use the constant from filter-utils actually, good catch

File: src/components/ProposalIcon.js (hunk 6 above)
Comment (fabriziovigevani) [reply src/components/ProposalIcon.js:8]: Ah actually we do have the Proposal and Suggestion colors, will use them 👍 
Though we don't have the one for the Decision

**Comment by rperez89** [comment]
LGTM @fabriziovigevani

**Review by rperez89 — APPROVED** [review]

**Pull request merged**